.github/workflows/szdiff.yml

```diff
@@ -49,3 +79,21 @@ jobs:
           skip_unchanged: true
           recreate: true
           message: ${{ env.loc_content }}
+
+  rebase:
+    name: Core Library Line Difference
+    permissions:
+      pull-requests: write
+    runs-on: ubuntu-latest
+    needs: checkbranch
+    if: needs.checkbranch.outputs.branchstat == 'true'
+    steps:
+      - name: Comment Rebase
+        continue-on-error: false
+        uses: marocchino/sticky-pull-request-comment@v2
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          skip_unchanged: true
+          message: |
+            This branch currently is behind tinygrad/master.
+            If there is no merge conflicts, please rebase.
```

Comment:
I think we can just mention that the current branch is behind, don't need the hint to rebase, maybe instead have it say that the line diff won't be shown until the branch is no longer behind?

Reply:
Yeah, it definitely sounds a lot nicer this way.

Reply:
Why hide line diffs? If there are no merge conflicts, the line diff is the same in both cases (rebased and not), isn't it? The only reason we might want to ask for a rebase is if the branch is too outdated to run tests from the master (to avoid getting failed tests after the PR is merged).

Reply:
> If there are no merge conflicts, the line diff is the same in both cases (rebased and not), isn't it?

Not necessarily. The bot doesn't track conflicts, but only counts the raw line difference between the given directory. So if there's other changes made which affect the count, it's going to be reflected even if the change is not from the PR branch.

Reply:
Well, maybe it's fine for now to show diff only when commits are rebased on the top of PR's target branch.

In the future for this case we could compare this with rebased version (to PR's target branch) or with pr's branch with N reverted commits (where N is number of commits in the PR).